types/applepayjs/index.d.ts

```diff
@@ -1035,6 +1059,26 @@ declare namespace ApplePayJS {
         newTotal: ApplePayLineItem;
     }
 
+    /**
+     * The attributes contained by the oncouponcodechanged callback function.
+     */
+    abstract class ApplePayCouponCodeUpdate {
```

Comment:
Many of the properties [in the documentation](https://developer.apple.com/documentation/apple_pay_on_the_web/applepaycouponcodeupdate) are missing from this definition.

Reply:
Done, thanks!
Could you please take a look?

It ended up being basically the same as the `interface ApplePayPaymentMethodUpdate`